ui-tests/cypress/support/pageObjects/components/mfeWidgets/MFEWidgetsPage.js

```diff
@@ -1,5 +1,68 @@
-import {DATA_TESTID, htmlElements, WebElement} from "../../WebElement.js";
+import { DATA_TESTID, htmlElements } from '../../WebElement';
 
-import Content from "../../app/Content.js";
+import Content from '../../app/Content';
+import AppPage from '../../app/AppPage';
+import MFEWidgetForm from './MFEWidgetForm';
+import Modal from '../../app/Modal';
 
-export default class MFEWidgetsPage extends Content {}
\ No newline at end of file
+export default class MFEWidgetsPage extends Content {
+
+  static WIDGET_ACTIONS = {
+    EDIT: 'WidgetListRow__menu-item-edit',
+    DELETE: 'WidgetListRow__menu-item-delete',
+  };
+
+  maincontent = `${htmlElements.div}[${DATA_TESTID}=list_ListWidgetPage_Grid]`;
+  rowlayout = `${htmlElements.div}[${DATA_TESTID}=list_ListWidgetPage_Row]`;
+
+  getContents() {
+    return this.get()
+      .children(this.maincontent);
+  }
+
+  getListArea() {
+    return this.getContents()
+      .find(this.rowlayout).eq(2);
+  }
+
+  getKebabMenuID(widgetCode) {
```

Comment:
Is this needed just for the following method? Why not just massing `widgetCode` to it?!

Reply:
this is needed by 2 sibling methods. so I would just stick this method in.

Reply:
shoot, i was looking at a different class. I removed it. should be okay now